the-content-guide/readme.md

```diff
@@ -7,14 +7,100 @@ The target audience are communication professionals. See it as a comprehensive g
 
 <img src="/images/contentcat.jpg" width="150">
 
-## File storage
+## Intro
 
-Each chapter should be it's own file. When you add a chapter, please provide the name of the file with a number. This number represents the order of the document chapters. After the number comes the chapters' name.
+Why do you need content marketing or branded content? Content marketing is a way to create a bond between a reader and a company. By creating content the reader will thoroughly enjoy, the reader will associate the brand with an awesome feeling and automatically think about that company first whenever searching for a product/service in that domain.
 
-## Pull requests
+Example: A solar cell company creates blogposts about:
+* how a solar cell work  
+* a comparison between green energy products
+* where is the future of solar cell power heading?
+* a foto collage of employees risking their lives on roofs
 
-Make your own branch. When working on the document, create a pull request for each document.
+So how do you attract readers and keep them engaged? There is an increasing demand for *'high quality content'*. But what is high quality content? What ingrediënts make a nice meal that people will remember? Better yet, how do you get people to ask for and share the recipe?
 
-## User story:
+This guide will describe what ingredients work well together and how you can best present the meal. It will allow you to attract your target audience and keep them maximally engaged. The guide will also explain how you can test your business metrics and how to reach your audience. Most of all, how to create content you can be proud of!
 
-To know what makes good content — As a communication professional — I can read a comprehensive guide about content creation.
+## Originality & authenticity
+
+Good content is original. [Content Farms](https://en.wikipedia.org/wiki/Content_farm) are no longer wanted in these times. Search engines, such as Google, are getting better and better at detecting these 'clickbait sites' and these sites are not received well. Original content is sure to be shared more often. Even more so, why would you say something when it's not worth saying?
+
+> You share content because you want to achieve a certain goal. You want it to be read, and you want the reader to learn something from it.
+
+Apart from originality high quality content also makes a certain promise about the credibility of the text. When sources are been used they should provide authority. Good sources are crucial and make sure that a text is taken seriously.
+
+Be a guardian of the originality and authenticity of your content piece. It's easier sad than done while we all scrape information from different kind of sources. When it's authority in people you interview, articles, books or your own judgement, it's all based on other work. The difference where you as an author come in, is how you handle these sources. Combine them to share new insights.
```

Comment:
"While we all scrape information...". "While" indicates that "It's easier said then done" is happening during the process of scraping information. "When" or "because" would be more accurate because they are words that put these two clauses in some kind of conceptual chronology rather indicating htat they happen at the same time. Make sense?